kuyruk/task.py

```diff
@@ -53,6 +54,9 @@ def __call__(self, *args, **kwargs):
         logger.debug("Task.__call__ args=%r, kwargs=%r", args, kwargs)
         self.send_to_queue(args, kwargs)
 
+    def subtask(self, args=(), kwargs={}, host=None):
```

Comment:
Mutable default parameters is something I usually avoid: http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments